receiver/opencensusreceiver/opencensus_test.go

```diff
@@ -365,3 +371,169 @@ func TestStartWithoutConsumersShouldFail(t *testing.T) {
 	mh := component.NewMockHost()
 	require.Error(t, r.Start(mh))
 }
+
+// TestOCReceiverTrace_HandleNextConsumerResponse checks if the trace receiver
+// is returning the proper response (return and metrics) when the next consumer
+// in the pipeline reports error. The test changes the responses returned by the
+// next trace consumer, checks if data was passed down the pipeline and if
+// proper metrics were recorded. It also uses all endpoints supported by the
+// trace receiver (unary and bidirectional streaming).
```

Comment:
Is `unary` available here?

Reply:
No, it is not. I will remove the reference to unary.